contracts/WooStakingVault.sol

```diff
@@ -257,6 +257,15 @@ contract WooStakingVault is ERC20, Ownable, Pausable {
         wooAccessManager = IWooAccessManager(newWooAccessManager);
     }
 
+    /// @notice Rescues random funds stuck
```

Comment:
@Note: this method only saves the irrelevant tokens just in case users deposited in mistake. It cannot transfer any of user staked tokens.